### DISCUSSION TITLE
Unpolished workflow for adding new pages

### DISCUSSION THREAD

**Discussion by bjornpiltz** [requarks/wiki#6855] [General]
Hi,
I'm new to Wiki.js and had a very pleasant first experience installing it and playing around a bit.

However, I have concerns when it comes to the daily workflow of adding content to the Wiki. I'm hoping there are some tricks I could use to simplify the process.

I'm used to Confluence and if I want to quickly add a page with meeting notes or something similar, I do the following.

### Confluence
| Action | Consequences |
| --- | --- |
| Click 'Add page' or press 'c' | A new draft is added in the current workspace, the keyboard input is at the Page title. |
| Type 'My Title' + Enter |  This sets the title of the page. The user is not exposed to any paths.  |
| Type 'My content' |     |
| Click 'Publish' or press 'ctrl+s' | A page with the title 'My title' is saved at the selected work space. |

Now, I hope I'm being a bit daft here, but it seems it is quite cumbersome to achieve the same in Wiki.js.

### Wiki.js
| Action | Consequences |
| --- | --- |
| Click 'New Page' | A dialog |
| Select folder i.e. '/dir' |     |
| Modify the path i.e. '/dir/my-title' | Note: you can't leave it at the default 'new-page' since this will open the page 'new-page' instead of creating a new page. |
| Click 'select' | A dialog |
| Select 'Visual Editor' | A dialog |
| Click in the 'Title' box |     |
| Enter 'My title\*' | Apparently the title is unrelated to the path of the document |
| Click 'ok' |     |
| Click in the document |     |
| Type 'My content' |     |
| Click 'Create' | A page with the title 'My title\*' is saved at /dir/my-title. |

### Other nitpicks
1. If 'dir' did not exist, it will show up in the sidebar under '/ sidebar.root' (sic) and can be selected there, although it has no effect. However clicking 'dir' in the breadcrumbs will lead to a 'Page not found' page.
2. If you edit an existing page, there is no option 'save and close editor'.

**Comment by NGPixel** [comment]
The workflow has been greatly simplified in the upcoming v3.
